docs/l10n.rst

```diff
@@ -0,0 +1,148 @@
+Localization (l10n)
+=====================
+
+These notes are primarily to support localizers working on localizations
+related to https://addons.mozilla.org/.
+
+Projects
+--------
+
+There are two core projects for AMO that require localization:
+
+    * `addons-server <https://github.com/mozilla/addons-server/>`_ (`addons-server on Pontoon`_)
+    * `addons-frontend <https://github.com/mozilla/addons-frontend/>`_ (`addons-frontend on Pontoon`_)
+
+Both of these projects use `Pontoon`_ for translations.
+
+Addons-server
++++++++++++++
+
+Addons-server is the current front-end for AMO, it is also the API
+backend and home to the developer hub and reviewer tools. It is written in
+Python using the Django Framework.
+
+Testing changes to addons-server localizations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The strings will be available on -dev or stage depending on when they were
+committed see `When can I expect to see my strings land?`_ for details.
+
+The relevant hosts for addons-server are as follows:
+
+    * dev https://addons-dev.allizom.org/
+    * stage https://addons.allizom.org/
+    * prod https://addons.mozilla.org/
+
+Addons-frontend
++++++++++++++++
+
+Addons-frontend is the new frontend for AMO, it will eventually replace the
+addons-server frontend for user-pages. It is written in JavaScript and uses
+React at it's core. Addons-frontend currently provides an amo app (amo.po).
+
+Testing amo localizations
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The strings will be available on -dev or stage depending on when they were
```

Comment:
the `-dev` again is odd, I'd remove the dash. I know it's used in the URL but it's odd to read.